File: SoloLearn_CodeCoach/Easy/Argentina/Argentina.cpp
#include <iostream>
using namespace std;

int main() {
    int Pesos;
    int Dollars;

    cin >> Pesos;
    cin >> Dollars;

    cout << (Pesos * 0.02 < Dollars ? "Pesos" : "Dollars") << endl;

    return 0;
}

File: SoloLearn_CodeCoach/Easy/Argentina/Argentina.java
import java.util.Scanner;

public class Argentina {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int pesos = input.nextInt();
        int dollars = input.nextInt();

        System.out.println(pesos * 0.02 < dollars ? "Pesos" : "Dollars");
    }
}
